junit5-extension/src/test/java/com/example/StubAnnotation.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package com.example;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ ElementType.FIELD, ElementType.METHOD, ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface StubAnnotation {
```

Comment:
Stub makes me think of stubbing code.  How about RuntimeAnnotationUsedForTestPurpose.

Reply:
I went with `RuntimeMarkerAnnotation` in dba0e5e0db10f70e0b6126117378ca2f9ddf494a as it was a bit shorter and conveyed the important point. 

I think the `ForTestPurposes` bit is implicit in being `com.example` in `src/test`.